src/app/core/services/attachment/attachment.service.ts

```diff
@@ -71,7 +71,7 @@ export class AttachmentService {
                 this.texts = data;
             });
         let opts = {
-            buttons: this.getActionSheetButtons(type)
+            buttons: this.getActionSheetButtons(profileImageData)
```

Comment:
here too

Reply:
Remove current photo option check whole profile image data is needed